Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -137,9 +137,8 @@ define([
                 tileContent.featurePropertiesDirty = false;
                 tile.lastStyleTime = 0; // Force applying the style to this tile
                 tileset._selectedTilesToStyle.push(tile);
-            } else if ((tile._selectedFrame !== frameState.frameNumber - 1)) {
+            } else if ((tile._selectedFrame < frameState.frameNumber - 1)) {
                 // Tile is newly selected; it is selected this frame, but was not selected last frame.
-                tile.lastStyleTime = 0; // Force applying the style to this tile
```

Comment:
This was originally added so that expired tiles would get restyled but was responsible for the first bug. Now `lastStyleTime` is reset above in the `// Refresh style for expired content` area.